src/client/Microsoft.Identity.Client/Internal/Requests/DeviceCodeRequest.cs

```diff
@@ -31,11 +31,8 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
 
             var client = new OAuth2Client(ServiceBundle.ApplicationLogger, ServiceBundle.HttpManager, null);
 
-            var deviceCodeScopes = new HashSet<string>();
-            deviceCodeScopes.UnionWith(AuthenticationRequestParameters.Scope);
-            deviceCodeScopes.Add(OAuth2Value.ScopeOfflineAccess);
-            deviceCodeScopes.Add(OAuth2Value.ScopeProfile);
-            deviceCodeScopes.Add(OAuth2Value.ScopeOpenId);
+            var deviceCodeScopes = new HashSet<string>(AuthenticationRequestParameters.Scope);
+            deviceCodeScopes.UnionWith(OAuth2Value.ReservedScopes);
```

Comment:
ReservedScopes are just offlineaccess + profile + openId

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/OAuth2/OAuthConstants.cs#L102

What is the actual issue you are seeing?